projectDocs/dev/developerGuide/developerGuide.t2t

```diff
@@ -1031,6 +1031,11 @@ For examples of how to define and use new extension points, please see the code
 || Type | Extension Point | Description |
 | ``Decider`` | ``decide_executeGesture`` | Notifies when a gesture is about to be executed, allowing other code to decide if it should be. |
 
+++ logHandler ++[logHandlerExtPts]
+
+|| Type | Extension Point | Description |
+| ``Action`` | ``_onErrorSoundRequested`` | Triggered every time an error sound needs to be played. This extension point should not be used directly but retrieved calling ``getOnErrorSoundRequested()`` instead. |
```

Comment:
While initially I agreed with @lukaszgo1's thinking that we shouldn't document a private variable like this, I think the way it is being done here makes clear how it is expected to be used. Documenting the actual variable which holds the extPoint is closer to the spirit of the rest of the Dev Guide chapter. Anyone who misuses it after reading the clear description @CyrilleB79 has written, is taking their own stupid risks.